asllib/doc/ASLTypingReference.tex

```diff
@@ -827,12 +1169,32 @@ \subsection{Example: TypingRule.TypeSatisfaction3.asl}
 
 \begin{emptyformal}
     \subsection{Formally}
+We denote that type $T$ type-satisfies type $S$ in an environment $E$ by $E \entails T \typesat S$.
+\[
+\begin{array}{c}
+\inferrule{E \entails T \subtype S }{E \entails T \typesat S}\\
+\\
+\inferrule{\entails T : \isanonymous \\ E \entails T \subtypesat S}{E \entails T \typesat S}\\
+\\
+\inferrule{\entails S : \isanonymous \\ E \entails T \subtypesat S}{E \entails T \typesat S}\\
+\\
+\inferrule{w_t,w_s \in \bitsconstraint \\ E \entails \tstruct(T)=\TBits(w_t, []) \\ E \entails \tstruct(S)=\TBits(w_s, \Ignore) \\ E \entails \llbracket w_s \rrbracket = \llbracket w_t \rrbracket}{E \entails T \typesat S}\\
+\end{array}
+\]
+
+\todocomment{
+ROMAN: define the evaluation of bitwidth length, which is used above.
+}
+
 \end{emptyformal}
 
 \subsection{Comments}
   Since the subtype relation is a partial order, it is reflexive. Therefore
   every type \texttt{T} is a subtype of itself, and as a consequence, every type \texttt{T}
-  type-satisfies itself.
+  type-satisfies itself. 
+\todocomment{  
+ROMAN: I disagree --- we defined subtyping to hold *only* on named types.
+}
```

Comment:
please make a proposal to address this as part of this PR

Reply:
The parser actually does the right thing already. We just need to fix the syntax in the LRM. I added a todo to my list of issues.